ingestion-edge/requirements.in

```diff
@@ -20,3 +20,7 @@ pytest-timeout==2.1.0
 pytest==7.2.0
 python-dateutil==2.8.2
 Sanic==20.12.7
+types-PyYAML
```

Comment:
no versions? probably actually better w/ the requirements.txt but inconsistent with the rest of the imports

Reply:
yeah, because they are types packages, i specifically decided not to include versions in the `.in` file. i'd actually like to remove unnecessary type limitations throughout this file, but that felt a bit out of scope.